cinema/models.py

```diff
@@ -50,8 +50,8 @@
     title = models.CharField(max_length=255)
     description = models.TextField()
     duration = models.IntegerField()
-    genres = models.ManyToManyField(Genre)
-    actors = models.ManyToManyField(Actor)
+    genres = models.ManyToManyField(Genre, blank=True)
+    actors = models.ManyToManyField(Actor, blank=True)
     image = models.ImageField(null=True, upload_to=movie_image_file_path)
```

Comment:
The `image` field in the `Movie` model uses a custom `upload_to` function, which is a good practice for organizing uploaded files. Ensure that the `movie_image_file_path` function handles all edge cases, such as filenames without extensions.